oak-doc/src/site/markdown/query/lucene.md

```diff
@@ -408,6 +408,14 @@ nodeScopeIndex
   explicit property definition provided then it would only be included if
   `nodeScopeIndex` is set to true.
 
+  Note : If an index definition consists of any property with nodeScopeIndex set to
+  true, then it will index the node name property for all the nodes (with nodeType
```

Comment:
What about "node name" instead of "node name property"? In my view, the node name isn't a "property" in the sense of a "JCR property".